<ui version="4.0" >
 <class>BodyForcesForm</class>
 <widget class="QWidget" name="BodyForcesForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>274</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <property name="layoutDirection" >
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground" >
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QGroupBox" name="groupBox" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font" >
      <font>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="title" >
      <string>Gravity</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="labelX" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>g&lt;sub>x&lt;/sub</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="lineEditX" >
            <property name="toolTip" >
             <string>Code_Saturne key word: GX</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" >
           <widget class="QLabel" name="labelUnitX" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>m/s&lt;sup>2&lt;/sup></string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="labelY" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>g&lt;sub>y&lt;/sub</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QLineEdit" name="lineEditY" >
            <property name="toolTip" >
             <string>Code_Saturne key word: GY</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" >
           <widget class="QLabel" name="labelUnitY" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>m/s&lt;sup>2&lt;/sup></string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QLabel" name="labelZ" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>g&lt;sub>z&lt;/sub</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1" >
           <widget class="QLineEdit" name="lineEditZ" >
            <property name="toolTip" >
             <string>Code_Saturne key word: GZ</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2" >
           <widget class="QLabel" name="labelUnitZ" >
            <property name="font" >
             <font>
              <weight>50</weight>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text" >
             <string>m/s&lt;sup>2&lt;/sup></string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>22</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Hydrostatic pressure</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QGridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="labelHPressure" >
            <property name="toolTip" >
             <string>Code_Saturne key word: IPHYDR</string>
            </property>
            <property name="text" >
             <string>Pressure interpolation in stratified flow:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" >
           <widget class="QRadioButton" name="radioButtonNo" >
            <property name="text" >
             <string>standard</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" >
           <widget class="QRadioButton" name="radioButtonYes" >
            <property name="text" >
             <string>improved</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>21</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
